sdk/cosmosdb/cosmos/package.json

```diff
@@ -45,6 +45,20 @@
     "webpack:dev": "webpack -d",
     "webpack": "webpack -p"
   },
+  "repository": {
```

Comment:
Was this intentionally changed?

Reply:
I believe VS Code overzealously reorganized the fields of this file.  I can revert the changes that weren't needed for the commit if necessary

Reply:
It would be good to undo and keep the changes scoped to core-http.

Reply:
There's one change to this file that is necessary, the bump of `tunnel` to `0.0.6` which is needed for `core-http` (AFAICT Rush forces me to make the versions consistent across the two packages).  I'll make sure that's the only line changed in this file.

Reply:
The change in this file is what is causing the CI failure at https://dev.azure.com/azure-sdk/public/_build/results?buildId=42933&view=logs. It is detecting changes to the package but no version number change to the cosmos package.

Reply:
Is it standard practice to bump the version number of a package before we're ready to ship an update?  If so I'm happy to bump the patch number of the cosmos package.

Reply:
I think it is generally a good practice to bump the version of the package whenever the package starts to diff from the published version. However I'm not entirely sure if bumping the version will fix this CI issue, it might still fail. I think it is only running in your PR because it detected changes to this file. 

@bsiegel what did you do when aligning other versions during your rush onboarding?

Reply:
Right now the npm version check doesn't block merge, @kpajdzik made that part of the `build` pipeline non-blocking for PR merges in PR #3355.  I do believe that bumping the version will resolve the issue but I was trying to resolve it since I wasn't sure whether that was the protocol in this case.  I'm happy to do whatever the team thinks is best!

Reply:
I didn't bump the version of any of our libraries. We usually do that right before we cut a release. It does get a little tricky to track the changes that have occurred since the last re,ease of a library (and thus figure out what SemVer bump we're required to do). There's a Rush workflow that can help us with that, but we're not using it just yet.